knowledge_repo/app/app.py

```diff
@@ -3,13 +3,15 @@
 import imp
 import logging
 import traceback
+import math
```

Comment:
Yeah... we should go through and do this consistently where it makes sense (here for example).